app/src/main/java/org/greenstand/android/TreeTracker/fragments/TreeHeightFragment.kt

```diff
@@ -40,6 +41,9 @@ class TreeHeightFragment : Fragment() {
 
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
 
+        requireActivity().toolbarTitle?.apply {
```

Comment:
Welcome back! And great PR!
We can reduce the code here by doing `requireActivity().toolbarTitle?.setText(textId)`